fitbenchmarking/results_processing/performance_profiler.py

```diff
@@ -83,6 +83,8 @@ def plot(acc, runtime, fig_dir):
             sorted_list = np.sort(value)
             step_values.append(np.insert(sorted_list, 0, 0.0))
 
+        labels = [key for key in acc.keys()]
```

Comment:
I think you can just pass in acc.keys(), and treat it as a list.

Reply:
Done